tasks/generate.py

```diff
@@ -146,3 +146,17 @@ def publish_release(ctx, version, user, pypi_name):
     print(' ', ','.join(emails))
     print()
     print('And announce it on twitter adding the #pytest hash tag.')
+
+
+@invoke.task(help={
+    'version': 'version being released',
+})
+def changelog_draft(ctx, version):
+    check_call(['towncrier', '--draft', '--version', version])
+
+
+@invoke.task(help={
+    'version': 'version being released',
+})
+def changelog_update(ctx, version):
```

Comment:
Perhaps you can have a single `changelog` command, with a `--draft` option instead of two commands?